src/stage/stage.ts

```diff
@@ -600,7 +600,7 @@ class Stage {
    */
   toggleFullscreen (element: HTMLElement) {
     if (!document.fullscreenEnabled && !document.mozFullScreenEnabled &&
-        !document.webkitFullscreenEnabled && !document.msFullscreenEnabled
+        !(document as any).webkitFullscreenEnabled && !document.msFullscreenEnabled
```

Comment:
it's a matter of taste there, but one can also legitimately use `//@ts-ignore` before the non standard statements

Reply:
sure -- didn't realize how many `@ts-ignore` comments were already in this codebase.

Reply:
As I said, just a matter of taste here ;-) It makes sense to leave it as is.